### PR TITLE
Feature/new constants

### PR DESCRIPTION
**Pull request by smic1909**
**Description**
Make use of Util.toIndentedString method 
- replace duplicate code 
- replace `\n` with constant

Ideally this PR will be merged after #355 since this branch has been created on the named one

### CODE REVIEW

**Comment by smic1909** [comment]
The idea was to separate these big changes into two PRs, that is why I added the comment to the description to first merge #355. I will revert the `Objects.isNull` changes as you suggested and close #355.

File: checkstyle.xml [hunk 1]
```diff
@@ -146,7 +146,7 @@
         <module name="EqualsHashCode"/>
         <!--<module name="HiddenField"/>-->
         <module name="IllegalInstantiation"/>
-        <module name="InnerAssignment"/>
+        <!-- <module name="InnerAssignment"/>-->
```
Comment (andrei-dediu) [inline checkstyle.xml:149]: Hey @smic1909 

What's the reasoning between commenting this check ?

Comment (smic1909) [reply checkstyle.xml]: Good catch. This was needed for some changes I did locally. Reverted.

**Comment by smic1909** [comment]
@andrei-dediu Will this PR be merged? It is already open for a while and I wonder why your the pending reviewer did not check the PR yet.

**Pull request merged**